tests/app/test_version_check.py

```diff
@@ -0,0 +1,110 @@
+from io import BytesIO
+from unittest.mock import patch
+
+from packaging.version import Version
+from requests import Response
+from snowflake.cli._app.version_check import _VersionCache, get_new_version_msg
+
```

Comment:
Missing tests:
- Error during get request not exit from app
- Message will not appear when cli_console is off

Reply:
Added both tests